src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,28 @@
+import { Link, useSearchParams } from 'react-router-dom';
+import cn from 'classnames';
+import { FC } from 'react';
+import { Person } from '../../types';
+
+interface Props {
+  person: Person;
+}
+
+export const PersonLink: FC<Props> = ({ person }) => {
+  const [searchParams] = useSearchParams();
+  const {
+    name,
+    sex,
+    slug,
+  } = person;
+
+  return (
+    <Link
+      to={`/people/${slug}?${searchParams}`}
+      className={cn('has-text-link', {
+        'has-text-danger': sex === 'f',
+      })}
```

Comment:
It would be very cool to put such strings into separate variables to avoid using magic data.
For example:
```
const FEMALE = 'f';
```
This applies to all code.